experiments-app/src/screens/ScrollViewEvents.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {
+  StyleSheet,
+  Text,
+  SafeAreaView,
+  ScrollView,
+  StatusBar,
+} from 'react-native';
+import { buildEventLogger } from '../utils/helpers';
+
+const handleOnMomentumScrollBegin = buildEventLogger('onMomentumScrollBegin');
+const handleOnMomentumScrollEnd = buildEventLogger('onMomentumScrollEnd');
+const handleOnScroll = buildEventLogger('onScroll');
+const handleOnScrollBeginDrag = buildEventLogger('onScrollBeginDrag');
+const handleOnScrollEndDrag = buildEventLogger('onScrollEndDrag');
+const handleOnScrollToTop = buildEventLogger('onScrollToTop');
+
+export function ScrollViewEvents() {
+  return (
+    <SafeAreaView style={styles.container}>
+      <ScrollView
+        style={styles.scrollView}
+        onMomentumScrollBegin={handleOnMomentumScrollBegin}
+        onMomentumScrollEnd={handleOnMomentumScrollEnd}
+        onScroll={handleOnScroll}
+        onScrollBeginDrag={handleOnScrollBeginDrag}
+        onScrollEndDrag={handleOnScrollEndDrag}
+        onScrollToTop={handleOnScrollToTop}
+      >
+        <Text style={styles.text}>
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
```

Comment:
Is that text long enough to make the screen larger than viewport? For this experiment to work well the content should take 2+ viewports.

Reply:
It is twice the viewport on my Pixel 6 device. I can make this text longer though.